pkg/project/auth/cache_test.go

```diff
@@ -0,0 +1,143 @@
+package auth
+
+import (
```

Comment:
I need the Fake client for PolicyBindings and Policy objects to actually support returning a Fake value.  @deads2k - is this something you were planning to add as part of your work?